src/main/resources/extracted/web/standalone/MySQL_configuration.php

```diff
@@ -23,14 +23,15 @@
 
 header('Content-type: application/json; charset=utf-8');
 
+$json = json_decode($content);
```

Comment:
the fact that the $json variable is decoded before the loginenabled check is not a perfect solution but overall i believe this is not a problem at all